report-deployment-status-to-slack/action.yml

```diff
@@ -56,32 +60,65 @@ runs:
       id: setup
       shell: bash
       run: |
-        case "${{ inputs.event }}" in
-          start)
-            SLACK_MESSAGE=":loading: Updating ${{ inputs.environment }} to \`${{ inputs.tag }}\` on ${{ inputs.project }}.
-            ${{ inputs.url }}
-            SHA: \`${{ inputs.commit-sha }}\`"
-            ;;
+        if [[ "${{ inputs.db_migration_detected }}" ]]; then
```

Comment:
As it is right now, this line is a string comparison, so it will always evaluate to `true`. The `else` block will never be run.

```
if [[ "false" ]]; then
  echo "This will run";
else
  echo "This will not run";
fi

# output: This will run
```

Reply:
I updated this.